js/CobrowseIO.js

```diff
@@ -114,11 +114,22 @@ export default class CobrowseIO {
 // the session.requested event is considered internal, it should
 // not be used outside these bindings
 CobrowseIO.addListener('session.requested', (session) => {
+  if (Platform.OS === 'ios' && CobrowseIO.handleFullDeviceRequest) {
+    CobrowseIONative.overwriteFullControlUI()
+  }
+
   CobrowseIO.handleSessionRequest(session)
 })
 
 CobrowseIO.addListener('session.updated', (session) => {
   if (session.isActive() && session.remote_control === 'requested') {
     CobrowseIO.handleRemoteControlRequest(session)
+  } else if (session.isActive() && session.full_device_state === 'requested') {
```

Comment:
Don't think there should be an `else` here? It's unlikely but possible for full device and remote control to be changed at the same time.

Reply:
updated, thanks.